source/Irrlicht/CBAWMeshWriter.cpp

```diff
@@ -317,4 +327,10 @@ namespace irr {
 			_ctx.offsets.push_back(!_ctx.offsets.size() ? 0 : _ctx.offsets.getLast() + _blobSize);
 		}
 
+		void CBAWMeshWriter::finalizeHeader(core::BlobHeaderV1 & _header, const void* _data, uint32_t _size)
```

Comment:
maybe make it a function of the BlobHeaderV* ?

Reply:
nice :+1: